CMakeLists.txt

```diff
@@ -28,6 +28,18 @@ elseif (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     add_definitions(-DMACOSX)
 endif()
 
+if (USE_BALTIMORE_CERT)
```

Comment:
If we want these exposed up via cmake, I think we should have option()'s for them grouped with the list below.  ANd then these if() checks should come after the option() stuff.

Everything else in this file that has in if(X) *without* the explicit option(), the X is a system thing like the compiler name or Linux.

Reply:
Rgr.